workup.gemspec

```diff
@@ -16,17 +16,15 @@ Gem::Specification.new do |spec|
   and minimal assumptions about the initial state of the machine.'
   spec.homepage      = 'https://github.com/cvent/workup'
 
-  spec.files         = Dir.glob('{lib,files,exe}/**/*', File::FNM_DOTMATCH)
+  spec.files         = %w(Gemfile LICENSE README.md workup.gemspec) + Dir.glob('{bin,files,lib}/**/*', File::FNM_DOTMATCH)
   spec.bindir        = 'bin'
-  spec.executables   = spec.files.grep(%r{^bin/}) { |f| File.basename(f) }
+  spec.executables   = %w(workup)
```

Comment:
I feel its better to include it this way because we want to explicitly add executables that get included